tests/conftest.py

```diff
@@ -66,3 +67,125 @@ def run_migrations() -> Iterator[None]:
             connection.execute(f"TRUNCATE TABLE IF EXISTS {database}.{table_name}")
 
     redis_client.flushdb()
+
+
+@pytest.fixture
+def convert_legacy_to_snql() -> Iterator[Callable[[str, str], str]]:
+    def convert(data: str, entity: str) -> str:
+        legacy = json.loads(data)
+
+        def func(value: Union[str, List[Any]]) -> str:
+            if not isinstance(value, list):
+                return value
+
+            children = ",".join(map(func, value[1]))
+            alias = f" AS {value[2]}" if len(value) > 2 else ""
+            return f"{value[0]}({children}){alias}"
+
+        def literal(value: Union[str, List[Any]]) -> str:
+            if isinstance(value, list):
+                return f"tuple({','.join(list(map(literal, value)))})"
+
+            try:
+                float(value)
+                return f"{value}"
+            except ValueError:
+                return f"'{value}'"
+
+        match_clause = f"MATCH ({entity[0].lower()}: {entity.capitalize()})"
+
+        selected = ", ".join(map(func, legacy.get("selected_columns", [])))
+        select_clause = f"COLLECT {selected}" if selected else ""
```

Comment:
I cannot remember anymore if we ever renamed this to SELECT.

Reply:
In a subsequent PR it will be renamed.